app/models/mixins/supports_feature_mixin.rb

```diff
@@ -86,6 +86,7 @@ module SupportsFeatureMixin
     :discovery                  => 'Discovery of Managers for a Provider',
     :evacuate                   => 'Evacuation',
     :launch_cockpit             => 'Launch Cockpit UI',
+    :admin_ui                   => 'Open Admin UI for a Provider',
```

Comment:
I'd suggest to add _oVirt_ somewhere in the description, and maybe also in the name of the feature.

Reply:
I just saw in other pull request that you are adding a `supports` mechanism to the provider. If that is going to be generic then you can disregard this comment.

Reply:
Yes, I've consulted this with @himdel and decided to make this feature generic - each Infra Provider would have the option to support the `admin_ui` product feature.